server/ai_process.go

```diff
@@ -33,6 +33,10 @@ const defaultUpscaleModelID = "stabilityai/stable-diffusion-x4-upscaler"
 const defaultAudioToTextModelID = "openai/whisper-large-v3"
 const defaultSegmentAnything2ModelID = "facebook/sam2-hiera-large"
 
+var errWrongFormat = fmt.Errorf("result not in correct format")
+
+var downloadResult = core.GetSegmentData
```

Comment:
Nit: why does it need to be a var? 

I think there is no reason right now, we may need it for some unit tests.

Reply:
This is copied implementation from transcoding side, https://github.com/livepeer/go-livepeer/blob/7ae8e2ce41eb2e46223b10aab47aa6bfab9eed73/server/broadcast.go#L51

Reusing `downloadSeg` seemed odd for the result download naming. Happy to update to any suggested naming.

Reply:
Could you just rename `GetSegmentData` to `DownloadResult` and get rid of this `var downloadResult` at all?

Reply:
I renamed the GetSegmentData function.  It created a number of changes in the transcoding side as well.  This is in a separate commit.

Reply:
I made some updates to transcoding side to add back in the `downloadSeg` var where needed for tests to pass.